test/unit/services/domain/test-capacity-date-range.js

```diff
@@ -1,16 +1,25 @@
 const expect = require('chai').expect
+const ValidationError = require('../../../../app/services/errors/validation-error')
+const ERROR_MESSAGES = require('../../../../app/services/validators/validation-error-messages')
+const FIELD_NAMES = require('../../../../app/services/validators/validation-field-names')
 const dateFormatter = require('../../../../app/services/date-formatter')
 const CapacityDateRange = require('../../../../app/services/domain/capacity-date-range')
+const CASELOAD_CAPACITY = require('../../../../app/constants/caseload-capacity')
 
 var capacityDateRange
 
 describe('services/domain/capacity-date-range', function () {
   const VALID_FROM_DAY = '01'
   const VALID_FROM_MONTH = '01'
-  const VALID_FROM_YEAR = '2017'
+  const VALID_FROM_YEAR = (new Date()).getFullYear() - 2
```

Comment:
Should this be some sort of configuration file rather then static `2` years back? If we are sure if it will never change then thats ok.